src/pages/ResetPasswordStep2.js

```diff
@@ -102,7 +102,11 @@ export default function ResetPasswordStep2({ api, email }) {
       </FormSection>
 
       <ButtonContainer className={"padding-medium"}>
-        <Button className={"full-width-btn"} onClick={handleResetPassword}>
+        <Button
+          type={"submit"}
+          className={"full-width-btn"}
+          onClick={handleResetPassword}
+        >
```

Comment:
@igawaclawska - why do we get so many formatting changes? don't we all have the same parameters for Prettier?